.travis.yml

```diff
@@ -3,6 +3,7 @@ sudo: required
 
 addons:
   chrome: stable
+  artifacts: true
```

Comment:
Artifacts in Travis are impossible to implement at the moment, so there's no need in this addon

Reply:
You right, I decide to remove this line.